src/styles.scss

```diff
@@ -2,60 +2,29 @@
 
 // Better style reset
 @import "~normalize.css/normalize";
-
 // A cleaner way to do breakpoints/media queries, http://breakpoint-sass.com/
 @import "~breakpoint-sass/stylesheets/breakpoint";
 @import "~system-font-css/_system-font.scss";
-
 // For the React DatePicker package.
 @import "~react-datepicker/dist/react-datepicker.css";
-// End libs
 
-/*
- * The `css-vars` mixins have been included here 
- * because they are utilized in the colors SCSS files.
- */
-
-// generates a collection of CSS vars
-@mixin css-vars($map, $prefix) {
-  $prefix: str-insert($prefix, "-", str-length($prefix) + 1);
-
-  @each $name, $color in $map {
-    --#{$prefix}#{$name}: #{$color};
-  }
-}
+// End libs
 
 // Order important
-@import "./styles/01-colors/_colors-brand";
-@import "./styles/01-colors/_colors-utility";
-@import "./styles/02-typography/_type-scale";
-@import "./styles/02-typography/_type-weight";
-@import "./styles/02-typography/_typefaces";
-@import "./styles/03-space/_space";
-@import "./styles/03-space/_space-inline";
-@import "./styles/03-space/_space-inset";
-@import "./styles/03-space/_space-stack";
-@import "./styles/base/_02-breakpoints";
-@import "./styles/base/_03-mixins";
-@import "./styles/base/_04-base";
-@import "./styles/base/_05-focus";
-@import "./styles/base/_card-grid";
+@import "./styles/base/_01-breakpoints";
+@import "./styles/base/_02-mixins";
+@import "./styles/base/_03-base";
+@import "./styles/base/_04-focus";
 @import "./styles/base/_place-holder";
-@import "./styles/base/_typography";
-@import "./components/Input/_Input.scss";
-@import "./components/Autosuggest/_Autosuggest.scss";
-
-// Used in `Input.tsx`
-.fieldset {
-  @include fieldset-reset;
-}
-
-.legend {
-  border: 0;
-  font-weight: 500;
-}
+@import "./styles/space/_space-inline";
+@import "./styles/space/_space-inset";
+@import "./styles/space/_space-stack";
 
-// Order not important
+// Legacy/deprecated components:
 @import "./components/CardEdition/_CardEdition.scss";
+@import "./components/Input/_Input.scss";
+
+// Components.
+@import "./components/Autosuggest/_Autosuggest.scss"; // Depends on _Input.scss
 @import "./components/DatePicker/_DatePicker.scss";
```

Comment:
There must be a way to write class names into the Chakra theme object.  In fact, I thought I was unknowingly doing that at one point.

Reply:
Yeah, we can in the theme object files (like the `breadcrumbs.ts`). I can do that now or make it another round. I'd like to also remove the `Input` and `CardEdition` scss files because they're legacy and deprecated and I think we can just completely remove those components...

Reply:
Let's use a new ticket to rework the styles for `DatePicker`.  And yes, I agree with regard to `Input` and `CardEdition` deprecation.